### PR TITLE
[WIP] V2

### PR DESCRIPTION
**Pull request by theimowski**
Creating this draft pull request to keep track of V2 branch CI

### CODE REVIEW

File: Content/.config/dotnet-tools.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "version": 1,
+    "isRoot": true,
+    "tools": {
+      "safe.tool": {
+        "version": "0.0.6",
+        "commands": [
+          "safe"
+        ]
```
Comment (Zaid-Ajaj) [inline Content/.config/dotnet-tools.json]: Any plans to add paket and fake as local dotnet tools for the template?

Comment (theimowski) [reply Content/.config/dotnet-tools.json]: yes, eventually

**Comment by isaacabraham** [comment]
Should this be closed?

**Comment by theimowski** [comment]
yup

**Pull request closed without merging**